pkg/internal/transform/route/cluster.go

```diff
@@ -61,11 +61,11 @@ func ClusterPath(path string) string {
 		if c == '/' {
 			nSegments++
 			if skip {
-				p[sPos] = '*'
+				p[sPos] = '#'
```

Comment:
To avoid introducing a breaking change, maybe we could do this value as configurable. Defaulting to `*`, we could add a `wildcard_char` property to the Routes configuration section.

Reply:
I think this is a good idea and something @marevers actually proposed in the issue. I think we should make it configurable, but for simplicity sake, we should only allow one character, otherwise the code will become more complex. It's OK to introduce breaking changes in the major versions, however we need to be cognisant that if we introduce this change with new default character we'll temporarily increase the cardinality for all customers when they switch.

Reply:
Alright, I can see if I can make it configurable (allowing a choice of a single character) while keeping `*` as the default option. That will remove the breaking change while still allowing the choice of something else.